tests/test_submit_message.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+from auth_helper import login_user, register_user
+from dotenv import load_dotenv
+
+from hushline import create_app, db
+from hushline.model import Message
+
+
+@pytest.fixture
+def client():
+    # Load environment variables from .env file or the equivalent
+    load_dotenv("env.sh")
```

Comment:
Using `dotenv` will override the vars we set in CI. we're already using `source env.sh` in local dev so this should never be necessary. Also we're creating the app fixture in every test file, and we want consistency, so we should just use `conftest.py`

Reply:
👍